docs/admin/kubeadm.md

```diff
@@ -362,6 +406,29 @@ For the gory details on how the tokens are implemented (including managing them
 outside of kubeadm) see the [Bootstrap Token
 docs](/docs/admin/bootstrap-tokens/).
 
+## Running kubeadm without an internet connection
```

Comment:
Personally I'd really prefer this and the cloud provider sections toward the end of the doc.  They are only applicable to a small number of users and having them in the middle makes the process seem harder than it really is.  Besides, anyone doing this will either do a dry run first or read the entire docs before acting.